database/execution_db_test.go

```diff
@@ -1,67 +1,26 @@
 package database
 
 import (
-	"io/ioutil"
-	"os"
 	"testing"
 
+	"github.com/cosmos/cosmos-sdk/codec"
+	"github.com/cosmos/cosmos-sdk/store"
+	"github.com/cosmos/cosmos-sdk/store/rootmulti"
 	"github.com/mesg-foundation/engine/execution"
 	"github.com/mesg-foundation/engine/hash"
 	"github.com/stretchr/testify/require"
+	dbm "github.com/tendermint/tm-db"
 )
 
-func db(t *testing.T, dir string) ExecutionDB {
-	db, err := NewExecutionDB(dir)
-	require.NoError(t, err)
-	return db
-}
-
-func TestFind(t *testing.T) {
-	dir, _ := ioutil.TempDir("", "TestFind")
-	defer os.RemoveAll(dir)
-	db := db(t, dir)
-	defer db.Close()
-	e := &execution.Execution{Hash: hash.Int(1)}
-	db.Save(e)
-	tests := []struct {
-		hash     hash.Hash
-		hasError bool
-	}{
-		{hash: e.Hash, hasError: false},
-		{hash: hash.Int(2), hasError: true},
-	}
-	for _, test := range tests {
-		execution, err := db.Find(test.hash)
-		if test.hasError {
-			require.Error(t, err)
-			continue
-		}
-		require.NoError(t, err)
-		require.NotNil(t, execution)
-		e, err := db.Find(execution.Hash)
-		require.NoError(t, err)
-		require.NotNil(t, e)
-	}
-}
+func TestExecutionDB(t *testing.T) {
+	db := NewExecutionDB(store.NewCommitMultiStore(dbm.NewMemDB()).(rootmulti.Store).GetStore(), codec.New())
```

Comment:
Why not using the levelDB store for now?
Like service:
```go
store, err := store.NewLevelDBStore(testdbname)
require.NoError(t, err)
db := NewServiceDB(store, codec.New())```

Reply:
I wanted to create cosmos store to get rid of leveldb in testing. In that case we can remove it completly once we switch to cosmos db.

Reply:
Look at cosmos_test in store. It already create a simple store:
```go
func newCosmosStore(t *testing.T) (*CosmosStore, func()) {
	store := NewCosmosStore(transient.NewStore())
	return store, func() {
		require.NoError(t, store.Close())
	}
}
```